Formula/c/chainhook.rb

```diff
@@ -0,0 +1,20 @@
+class Chainhook < Formula
+  desc "Reorg-aware indexing engine for the Stacks & Bitcoin blockchains"
+  homepage "https://github.com/hirosystems/chainhook"
+  url "https://github.com/hirosystems/chainhook.git",
+      tag:      "v1.4.1",
+      revision: "9d9f0d66dc32c7680fdabd3dba2ecf2847dd2b2a"
+  license "GPL-3.0-only"
+  head "https://github.com/hirosystems/chainhook.git", branch: "develop"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "chainhook-install", "--root", prefix.to_s
```

Comment:
also use std_cargo_args in here as well, see some build change in https://github.com/Homebrew/homebrew-core/pull/168061